api/proto/teleport/accesslist/v1/accesslist.proto

```diff
@@ -84,6 +84,9 @@ message AccessListSpec {
 
   // owner_grants describes the access granted by owners to this access list.
   AccessListGrants owner_grants = 11;
+
+  // member_count is the number of members in the access list.
+  int32 member_count = 12;
```

Comment:
This doesn't fit in the `spec`, the number of members at a specific time is not at all part of the intended specification of the access list; rather, this seems to be a perfect fit for the `status` top level field.

Quoting [RFD 0153](https://github.com/gravitational/teleport/blob/master/rfd/0153-resource-guidelines.md#defining-a-resource):
> If a resource has properties that are required to be modified dynamically by Teleport, a separate `status` field should be added to the resource to contain them. These fields will be ignored by IaC tools during their reconcilliation.